<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="spinBox_count">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>256</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deleteDublicate">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Удаление дубликатов</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>116</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_min">
       <property name="text">
        <string>Минимальное</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_max">
       <property name="text">
        <string>Максимальное</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_avg">
       <property name="text">
        <string>Среднее</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>65</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_min">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_max">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_avg">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>104</width>
      <height>175</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_bubble">
       <property name="text">
        <string>Пузырьковая</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_gnom">
       <property name="text">
        <string>Гномья</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_comb">
       <property name="text">
        <string>Расческа</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_quick">
       <property name="text">
        <string>Быстрая</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_monkey">
       <property name="text">
        <string>Обезъянья</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>214</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_search">
       <property name="text">
        <string>Поиск:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_searchValue"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="search">
       <property name="text">
        <string>Поиск</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_random">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Заполнить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_searchRows">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_count">
       <property name="text">
        <string>Кол-во:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_searchCount">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu8_10">
    <property name="title">
     <string>8-10</string>
    </property>
   </widget>
   <addaction name="menu8_10"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
